README.md

```diff
@@ -1,2 +1,45 @@
+
+Access the docs [here](https://docs.flrns.domains/).
+
+
+## Navigating the Monorepo
+
+- [Front End](./apps/next) - nextJS environment for building the [FNS dApp](https://app.flrns.domains/).
+
+- [Smart Contracts](./apps/forge) - Foundry environment for testing, compiling, and development of the smart contract protocol.
+
+
+## How it Works
+
+### FNS App
+
+To get started, navigate to the FNS app at app.flrns.domains using a Web3 compatible browser and a Flare browser extension wallet such as Metamask (see a list of compatible wallets here). You can utilize the search function to browse through domain name options and determine which domains are currently available. Additionally, the app allows you to view the details of previously registered domain names, including the address that holds ownership, the address to which they resolve, and the expiration date of their registration.
+
+### Registering
+
+Once your browser wallet has been authorized to connect to the FNS app, you will be able to initiate the registration process for a .flr domain. Before submitting your registration request, ensure that your wallet contains enough FLR cryptocurrency to cover the registration fee and any associated gas fees. 
+By default, FNS requires a two-step registration process to protect all users from front-running attacks.
+Once you have confirmed the availability of a desired domain address and chosen the desired registration duration, click the "commit" button.
+
+After signing the initial transaction with your browser extension wallet, you must wait a few minutes while the contracts prevent you from falling prey to a front-running attack. Once the registry approves your request, you will be prompted to finalize your registration by signing another transaction to register, which will likely entail the highest gas fee due to the computation involved.
+Once the second transaction is processed, you will officially become the owner of a new .flr domain; however, additional configuration steps may be necessary before the domain is fully functional.
+
+### Resolve Domain Address
+
+Once you have successfully registered your .flr domain, the next step is to assign it to a Flare wallet or smart contract address of your choice. This can be accomplished by setting the ReverseRegistrar to point to your desired Flare Wallet Address on the My Account page.
+You have the flexibility to choose whether the domain resolves to your own Flare wallet address or to an alternate wallet address or smart contract address. Once you have specified the desired address in the domain records, any transactions sent to the .flr domain will be automatically routed to the corresponding address via the resolver. To do this, you must be using a wallet provider that supports FNS domain name look-ups.
```

Comment:
1. "You have the flexibility to choose whether the domain resolves to any valid address on the Flare network including a personal wallet address or even a smart contract address."
2. "... will automatically be routed to the corresponding address ..."